### PR TITLE
CB-10481 Update the docs according to windows8 deprecation

### PR DESCRIPTION
**Pull request by daserge**
[Jira issue](https://issues.apache.org/jira/browse/CB-10481)

### CODE REVIEW

**Comment by daserge** [comment]
@nikhilkh, @omefire, @sarangan12, could you please take a look if possible?

**Comment by nikhilkh** [comment]
@rakatyal to review

File: www/docs/en/dev/guide/platforms/win8/index.md [hunk 1]
```diff
@@ -49,37 +49,23 @@ on how to support IE along with comparable WebKit browsers.
...
-  or Visual Studio 2013.  Visual Studio 2015 is not able to build Windows 8.0 apps.
-
-To develop apps for Windows 8.0 and 8.1 (including Windows Phone 8.1):
-
-- Windows 8.1 or Windows 10, 32 or 64-bit _Home_, _Pro_, or _Enterprise_ editions,
-  along with 
-  [Visual Studio 2013 Express](http://www.visualstudio.com/downloads)
-  or higher. An evaluation version of Windows 8.1 Enterprise is
+- A Windows 8.1, 32 or 64-bit machine (_Home_, _Pro_, or _Enterprise_ editions) 
+  with minimum 4 GB of RAM along with [Visual Studio 2015](http://www.visualstudio.com/downloads)
```
Comment (sarangan12) [inline www/docs/en/dev/guide/platforms/win8/index.md]: Can you make this reference link? And all the other links in the page where you are specifically making the changes.

**Comment by sarangan12** [comment]
LGTM

**Comment by rakatyal** [comment]
Can we change the folder name for guides from win8 to windows as well? www/docs/en/dev/guide/platforms/win8/index.md  becomes www/docs/en/dev/guide/platforms/windows/index.md

**Comment by nikhilkh** [comment]
@rakatyal  I think we should do that as part of a single page guide that you will take up as it requires to setup 301 re-directs.

**Comment by rakatyal** [comment]
Sounds good. LGTM!

File: www/docs/en/dev/guide/platforms/win8/index.md (hunk 1 above)
Comment (daserge) [inline www/docs/en/dev/guide/platforms/win8/index.md]: @sarangan12, could you please clarify which link do you mean?
There is a link to the Windows evaluation version below:

>  An evaluation version of Windows 8.1 Enterprise is available from the [Microsoft Developer Network](http://msdn.microsoft.com/en-US/evalcenter/jj554510).

Comment (daserge) [inline www/docs/en/dev/guide/platforms/win8/index.md]: Oh, I see it's not working now - I'll update it to [https://technet.microsoft.com/evalcenter/hh699156.aspx](https://technet.microsoft.com/evalcenter/hh699156.aspx).

**Comment by daserge** [comment]
@sarangan12, updated the links, please take a look.

**Comment by sarangan12** [comment]
LGTM

**Comment by omefire** [comment]
Should we not also remove Win8 from other languages ? 
i.e: https://github.com/apache/cordova-docs/tree/95c3b8c8623ea2ac748ceb25f66a5a272c028238/www/docs/zh/dev/guide/platforms/win8

**Comment by daserge** [comment]
@omefire, I thought that translations are handled (semi)automatically using Crowdin, i.e. the `en/dev` version updates are translated and a snapshot then copied to www/docs/lan/v.e.r?
Am i wrong?

**Pull request merged**